apps/glusterfs/app_device.go

```diff
@@ -217,11 +217,10 @@ func (a *App) DeviceDelete(w http.ResponseWriter, r *http.Request) {
 		device, err = NewDeviceEntryFromId(tx, id)
 		if err == ErrNotFound {
 			http.Error(w, err.Error(), http.StatusNotFound)
-			return err
+			return logger.Err(err)
```

Comment:
Some of these do not need to be logged.  For example, "Not found" should not be logged as "ERROR".  If anything, it can be logged as WARNING or INFO, but not ERROR.

:-1: